graphql/src/main/java/com/linecorp/armeria/server/graphql/ExecutionResultSubscriber.java

```diff
@@ -67,7 +68,7 @@ public void onNext(ExecutionResult executionResult) {
                 protocol.sendGraphqlErrors(executionResult.getErrors());
                 subscription.cancel();
             }
-        } catch (JsonProcessingException e) {
+        } catch (JsonProcessingException | ClosedStreamException e) {
```

Comment:
Regardless of the exception type, it would be safe to cancel.
```suggestion
        } catch (Throwable e) {
```